src/validation/validate.ts

```diff
@@ -55,6 +56,11 @@ export function validate(options: BeachballOptions, validateOptionsOverride?: Pa
       console.log(options.changehint);
       process.exit(1);
     }
+
+    if (options.keepChangeFiles && areChangeFilesDeleted(options)) {
+      console.error('ERROR: Change files must not be deleted!');
```

Comment:
Can we output what files were deleted? Or does that already happen in the areChangeFilesDeleted flow?

Reply:
No, deleted files are not printed anywhere. I can fix that quickly.